package.json

```diff
@@ -28,13 +31,8 @@
     "prettier:all": "yarn workspaces foreach run prettier",
     "lint:all": "yarn workspaces foreach run lint",
     "build:ci:cloudflare": "bash ./utils/build_cloudflare.sh",
-    "build:ci:netlify": "yarn build:all"
-  },
-  "husky": {
-    "hooks": {
-      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS",
-      "pre-commit": "lint-staged"
```

Comment:
I assume that this is needed for `lint-staged` to run?

Reply:
No, script command is run from https://github.com/ubiquity/ubiquity-dollar/blob/8f295b7eb0983254b26b5f3dab9f480a88831b96/.husky/pre-commit

I tried to remove husky config from `package.json` and everything worked fined so I decided not to keep commands in 2 places: `package.json` and `.husky` folder thus leaving only `.husky` folder.

Reply:
Awesome work!